### PR TITLE
Render Coral Talk comments

### PR DESCRIPTION
**Pull request by ker-an**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
+	"scripts": {
+		"demo": "obt demo && http-server demos/local"
```
Comment (JakeChampion) [inline package.json]: We have a flag which will run a server -- `obt demo --run-server`

Comment (ker-an) [reply package.json]: Oh perfect! I'll delete the script 😄

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,4 +33,4 @@ test:
           tags:
             only: /^v.*/
           branches:
-            ignore: /.*/
\ No newline at end of file
+            ignore: /.*/
```
Comment (GlynnPhillips) [inline .circleci/config.yml]: We need to pull in the latest changes in `master` to enable releases in the circle config

File: src/js/comments.js [hunk 3]
```diff
@@ -6,9 +6,22 @@ class Comments {
...
+	/**
+	 * Render comment instances and store them on the `_comments` property.
+	 *
+	 * @access private
+	 * @returns {void}
+	 */
+	_renderComments() {
+		const scriptElement = document.createElement('script');
+		scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
+		scriptElement.setAttribute('onload', `Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'), {talk: 'https://ft-next-talk-spike.herokuapp.com'})`);
```
Comment (tbergmen) [inline src/js/comments.js]: Aren't we supposed to pass a JWT token to the component at this stage?

Comment (ker-an) [reply src/js/comments.js]: Oh, I'm not sure! @GlynnPhillips, any thoughts?

File: .circleci/config.yml (hunk 2 above)
Comment (ker-an) [reply .circleci/config.yml]: Done 😄

File: src/js/comments.js (hunk 3 above)
Comment (ker-an) [reply src/js/comments.js]: In real life Glynn said: yes, we it does need to be passed in eventually but that can be done at a later time. We are still able to view the comments box - just can't post a comment without the token.

**Pull request merged**